askama_rocket/src/lib.rs

```diff
@@ -6,11 +6,11 @@ pub use rocket::request::Request;
 use rocket::response::Response;
 pub use rocket::response::{Responder, Result};
 
-pub fn respond<T: Template>(t: &T, ext: &str) -> Result<'static> {
+pub fn respond<'r, 'o, 'r, T: Template>(t: &'r T, ext: &'r str) -> Result<'o> {
```

Comment:
Uh, is it actually useful/necessary to have two lifetimes with the same name? That seems potentially confusing.

Reply:
Ah! It was from upstream. I have removed the subsequent lifetime.